tests/test_core.py

```diff
@@ -1974,13 +1974,11 @@ def test_biggerswitch(self):
 59899: 598995989959899
 Success!''')
 
-  @no_wasm_backend('no implementation of computed gotos')
```

Comment:
Isn't this still unavailable?

Reply:
No, @dscuff fixed things in https://bugs.llvm.org/show_bug.cgi?id=42498